libavfilter/dnn/dnn_backend_openvino.c

```diff
@@ -375,6 +376,11 @@ static DNNReturnType init_model_ov(OVModel *ov_model, const char *input_name, co
         ctx->options.nireq = av_cpu_count() / 2 + 1;
     }
 
+    if (ctx->options.async != 0 && ctx->options.async != 1) {
```

Comment:
do we need this if? The allowed values are 0 and 1.

    { "async",           "use DNN async inference",       OFFSET(options.async),           AV_OPT_TYPE_BOOL,   { .i64 = 1 },     0,       1, FLAGS },

Reply:
Oh right. Thanks for the catch.